src/_pytest/python.py

```diff
@@ -138,6 +138,16 @@ def pytest_configure(config: Config) -> None:
     )
 
 
+@final
+class PytestUnhandledCoroutineError(UsageError):
+    """An unraisable exception resulted in an error.
```

Comment:
This docstring is wrong, looks like it was copied out of the unraisablehook plug-in